src/collective/nitf/upgrades/v1008/__init__.py

```diff
@@ -12,7 +12,10 @@ def fix_collections(context):
     logger.info('About to update {0} objects'.format(len(collections)))
     for col in collections:
         obj = col.getObject()
-        query = obj.getQuery()
+        try:
+            query = obj.getRawQuery()
```

Comment:
@rodfersou then you should know when getRawQuery is available and from where it came; I was not able to find it on the whole Plone code base.

also, that explanation should be on the code as comments.